bin/live/pycbc_live_single_trigger_fits

```diff
@@ -266,7 +266,7 @@ for filename in files:
         # If we are clustering, take the max sngl_ranking value
         if args.cluster:
             max_idx = sngls_value.argmax()
-            triggers_cut = triggers_da.select(max_idx)
+            triggers_da = triggers_da.select([max_idx])
```

Comment:
why is it better to index on a list containing max_idx rather than max_idx itself?

Reply:
If we use `max_idx` on its own, the returned DictArray's `data` is a dictionary of floats, rather than arrays, which breaks the addition of the triggers [here](https://github.com/gwastro/pycbc/blob/master/bin/live/pycbc_live_single_trigger_fits#L272)

Reply:
OK, I was going to request a comment to explain this. Can add that as a separate commit I guess

Reply:
it's a list isn't it? if so would it not be simpler to always initialize the list (`events[ifo] = []`) rather than have a dictionary that may have an entry or may not?

Reply:
> it's a list isn't it? if so would it not be simpler to always initialize the list (`events[ifo] = []`) rather than have a dictionary that may have an entry or may not?

triggers_da is a DictArray - this needs the keys from the triggers to be known before going into the loop over files. I suppose we could open file 0 and read them initially (something like events = {ifo: DictArray(data={k: [] for k in keys})}`, but this way is a bit cleaner to code IMO

Reply:
Oh, I see (I think). Is this not a shortcoming of DictArray that it allows `data` to be a type that breaks its own `__add__` method?